### PR TITLE
Testing howto

### PR DESCRIPTION
**Pull request by josefinnilsson**

### CODE REVIEW

File: TEST.txt [hunk 1]
```diff
@@ -0,0 +1,53 @@
+Four different scenarios can be tested with our DoS mitigation tool dostroy:
+1. Rate limiting
+2. Slowloris prevention
+3. RUDY prevention
+4. Generic error handling
+
+Requirements:
+- Node.js installed (≥ v8.15.0).
+- npm installed (≥ v6.10.2).
+- Preferrably two machine to run the tests on as behaviour can't be guaranteed when the attacker and target is the same computer.
```
Comment (Samuelhertzberg) [inline TEST.txt]: Preferrably -> Preferably

Comment (Samuelhertzberg) [reply TEST.txt]: machine -> machines

File: TEST.txt [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+4. Wait a few seconds to see that you're now able to enter the page again.
+5. On computer 2, go to the flood-dos repository and run it with the following command: ./bashFlood [IP of computer 1] 3000 500 in order to send 500 requests to the example server. As you can see in your terminal, the requests will be blocked.
+
+Testing Slowloris:
+1. On computer 1, deactivate slowloris prevention by changing the config in example-server/server/server.js to say slowloris: false.
+2. Start the server.
+3. On computer 2, go to the slowloris repository. In the file index.js (line 6), change the variable host to the IP address of computer 1.
+4. Start the application by running node index.js. You will be asked for the port (it's 3000) and the number of connections you want to start (about 150 should work).
+5. Try to visit the server in your browser. It should now be under attack and not be able to visit.
+6. On computer 1, activate slowloris prevention by chaning the config object to slowloris: true. Start the server again.
```
Comment (Samuelhertzberg) [inline TEST.txt]: chaning -> changing

File: TEST.txt [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+2. Start the server.
+3. On computer 2, go to the rudy repository. In the file index.js (line 8), change the variable host to the IP address of computer 1 including port 3000.
+4. Start the application by running node index.js.
+5. Try to visit the server in your browser. It should now be under attack and not be able to visit.
+6. On computer 1, activate RUDY prevention by changing the config object to rudy: true. Start the server again.
+7. On computer 2, run the rudy tool again.
+8. Visit the server to see that the slowloris attack is now prevented and you're able to visit the page.
+
+Testing Generic error handling:
+1. On computer 1, deactive error handling by removing line 48 in example-server/server/server.js.
```
Comment (Samuelhertzberg) [inline TEST.txt]: deactive -> deactivate

File: TEST.txt (hunk 1 above)
Comment (josefinnilsson) [reply TEST.txt]: I fixed all of the spelling errors in the second commit :)

**Pull request merged**